src/components/createbook.css

```diff
@@ -0,0 +1,55 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  background-color: white;
+  border-radius: 0.5rem;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.forminput {
+  display: flex;
+  gap: 1rem;
+  justify-content: space-between;
+}
+
+.Lesson-Panel1 {
+  display: flex;
+  flex: 4;
+  padding: 1rem;
+  border-radius: 4px;
+  border: solid 1px #e6e6e6;
+  background-color: white;
+}
+
+.Lesson-Panel2 {
+  display: flex;
+  flex: 2;
+  padding: 1rem;
+  border-radius: 4px;
+  border: solid 1px #e6e6e6;
+  background-color: white;
+}
```

Comment:
- [ ] These two inputs share the same values for every properties except for the flex, please refactor your code by following the DRY principle.

You might consider doing it this way 👇🏽 

```
.Lesson-Panel1 , .Lesson-Panel2 {
  display: flex;
  padding: 1rem;
  border-radius: 4px;
  border: solid 1px #e6e6e6;
  background-color: white;
}

.Lesson-Panel1 {
 flex: 4;
}

.Lesson-Panel2 {
  flex: 2;
}

```